about.html

```diff
@@ -0,0 +1,169 @@
+        <div class="mob-menu">
+          <i class="fa-sharp fa-solid fa-xmark close_btn"></i>
+          <ul>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="index.html">Program</a></li>
+            <li><a href="about.html">Join</a></li>
+            <li><a href="about.html">Sponsor</a></li>
+            <li><a href="about.html">News</a></li>
+          </ul>
+        </div>
+      </nav>
+      <nav class="desk_nav">
+        <div class="top-nav">
+          <i class="fa-brands fa-facebook-f"></i>
+          <i class="fa-brands fa-twitter"></i>
+          <ul>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+          </ul>
+        </div>
+        <div class="desktop-nav">
+          <a href="index.html"
+            ><img src="assets/NAMMlogo-2.jpg" alt="NAMM Logo"
+          /></a>
+          <ul>
+            <li><a href="about.html">About</a></li>
+            <li><a href="index.html">Program</a></li>
+            <li><a href="about.html">Join</a></li>
+            <li><a href="about.html">Sponsor</a></li>
+            <li><a href="about.html">News</a></li>
+            <li class="connect">
+              <a href="https://www.namm.org/contact">Get Connected</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+      <section class="intro-section">
+        <div class="name_title_abt">
+          <h3 class="intro-text-abt">"Hello! Musical World"</h3>
+          <h1 class="mn-title-abt">
+            <span> NAMM Show </span>
+          </h1>
+          <p class="title_desc-abt">
+            <span
+              >(National Association of Music Merchants) <br />
+              @East Africa
+            </span>
+          </p>
+          <div class="desc-text-abt">
+            <p>
+              The NAMM show is an annual event that is organized by the National
+              Association of Music Merchants (NAMM), who describe it as "the
+              industry's largest stage, uniting the global music, sound and
+              entertainment technology communities". It is one of the largest
+              music product trade shows in the world. <br />
+              <br />
+              The event attracts numerous Legendary and famous musicians, many
+              of whom are endorsed by exhibitors and come to promote their own
+              signature models and equipment.
+            </p>
+          </div>
+          <div class="time_place-abt">
+            <p>
+              Please contact on our website form for any further questions about
+              The NAMM Show East Africa 2023!
+            </p>
+            <br />
+            <h4>
+              <a href="https://www.namm.org/contact"
+                >https://www.namm.org/contact</a
+              >
+            </h4>
+          </div>
+        </div>
+      </section>
```

Comment:
Just as I suggested for the index page, your intro section should be wrapped in the `main` tag, kindly make reference to the comment for clarity😉